src/h_assets/__init__.py

```diff
@@ -1 +1,2 @@
-from h_assets.assets import Environment, assets_view
+from h_assets.environment import Environment
+from h_assets.view import assets_view
```

Comment:
I have the same query about the `view` and `environment` modules as for `cached_file` in the previous PR (ie. should these have an underscore prefix?).

Reply:
No, as these are the exported members of the library they are the public interface. So in order for the underscore to convey meaning to the developer it must indicate something. Otherwise in libraries arguably every file would have an underscore on it, not mean very much, and be kind of ugly.

So on the whole if:

* You are the thing being exported in your package you don't have an underscore
* If your parent is private with an underscore you don't need one (you are "public" locally, but not globally)
* If _everything_ is private in a directory, you don't need one (similar to the above) as it conveys no information as the purpose is to differentiate between things. If everything is the same there's no differentiation.